src/main/java/uk/gov/hmcts/reform/judicialapi/elinks/util/JobStatus.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.judicialapi.elinks.util;
+
+public enum JobStatus {
+
+    IN_PROGRESS("IN_PROGRESS"),
+
+    FAILED("FAILED"),
+
+    SUCCESS("SUCCESS"),
+    
+    FILE_LOAD_FAILED("FILE_LOAD_FAILED");
```

Comment:
This flag should be removed. No requirement of FILE_LOAD|_FAILED. Please check

Reply:
Needful done @manukundloo-hmcts